packages/core/admin/admin/src/content-manager/components/RelationInput/components/RelationList.js

```diff
@@ -19,34 +19,34 @@ const ShadowBox = styled(Box)`
 
   &:before {
     /* TODO: as for DS Table component we would need this to be handled by the DS theme */
-    content: ${({ overflow }) =>
-      overflow === 'top-bottom' || overflow === 'top' ? "''" : undefined};
+    content: ${({ overflowDirection }) =>
+      overflowDirection === 'top-bottom' || overflowDirection === 'top' ? "''" : undefined};
     background: linear-gradient(rgba(33, 33, 52, 0.1) 0%, rgba(0, 0, 0, 0) 100%);
     top: 0;
   }
 
   &:after {
     /* TODO: as for DS Table component we would need this to be handled by the DS theme */
-    content: ${({ overflow }) =>
-      overflow === 'top-bottom' || overflow === 'bottom' ? "''" : undefined};
+    content: ${({ overflowDirection }) =>
+      overflowDirection === 'top-bottom' || overflowDirection === 'bottom' ? "''" : undefined};
     background: linear-gradient(rgba(0, 0, 0, 0) 0%, rgba(33, 33, 52, 0.1) 100%);
     bottom: 0;
   }
 `;
 
-export const RelationList = ({ children, ...props }) => {
+export const RelationList = ({ children, overflow, ...props }) => {
   return (
-    <ShadowBox {...props}>
+    <ShadowBox overflowDirection={overflow} {...props}>
       <Stack spacing={1}>{children}</Stack>
     </ShadowBox>
   );
 };
 
 RelationList.defaultProps = {
-  overflow: undefined,
+  overflow: '',
```

Comment:
👀 :)

Reply:
I was wondering about this one, since we allowed empty string as a good value, do you think `undefined` is better? or I'm missing something else 👀

Reply:
No I was wondering, because you renamed the prop :)

Reply:
ah true but I renamed it only for `ShadowBox`, does that sound good to you?

<img width="516" alt="image" src="https://user-images.githubusercontent.com/71838159/189901677-dd1da600-d8c2-414d-ab25-f0bda975ee7a.png">

Reply:
Ah, if that fixes the issue I'm good with it.

Reply:
I changed it only for ShadowBox because Box was integrating the CSS property 🤓